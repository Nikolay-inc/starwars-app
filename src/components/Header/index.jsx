import styles from "./styles.module.css";

// basic layout header
const Header = () => {
  return (
    <header className={styles.header} data-testid="header">
      StarWars Heroes
    </header>
  );
}

export default Header;