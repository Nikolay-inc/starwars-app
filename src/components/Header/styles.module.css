.header {
  background: #000;
  height: 70px;
  text-align: center;
  font-size: 32px;
  color: #fff;
  padding: 15px 0;
}