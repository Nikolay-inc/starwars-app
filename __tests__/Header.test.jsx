import { render } from '@testing-library/react';
import Header from '../src/components/Footer';

describe('Header component', () => {
  test('renders correctly', () => {
    const { getByText } = render(<Header />);
    const headerElement = getByText('StarWars');
    expect(headerElement).toBeInTheDocument();
  });

  test('matches snapshot', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
