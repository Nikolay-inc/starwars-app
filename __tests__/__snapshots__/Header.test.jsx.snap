// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<div>
  <footer
    class="footer"
    data-testid="footer"
  >
    <span>
      StarWars
    </span>
  </footer>
</div>
`;
